docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "",
```

Comment:
please create a discussion thread in this repo for your vector db

Reply:
Done